apps/src/templates/teacherHomepage/RecentCoursesCollapsible.jsx

```diff
@@ -26,7 +26,7 @@ const RecentCoursesCollapsible = React.createClass({
         linkText={i18n.viewAllCourses()}
         link="https://studio.code.org/"
       >
-        {courses ? (
+        {courses.length > 0 ? (
```

Comment:
this shouldn't be necessary, as `[].map` will also lead to an empty array. You could also reduce a level of indentation by doing:
```
(courses || []).map((course, index) => {
```
though whether or not you want to is up to you.